README.md

```diff
@@ -165,31 +112,34 @@ Can be combined with `eo:bands`'s [`Band Object`][stac-ext-eo-band-obj].
 | tag         | string  | Tag of the image.                                     |
 | working_dir | string  | Working directory in the instance that can be mapped. |
 | run         | string  | Running command.                                      |
-| gpu         | boolean | True if the docker image requires a GPU.              |
+| accelerator         | boolean | True if the docker image requires a custom accelerator (CPU,TPU,MPS).              |
+
+### Output
 
-### Outputs Object
+| Field Name               | Type                    | Description                                                                                                                                                                        |
+|--------------------------|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| task                     | [Task Enum](#task-enum) | Specifies the Machine Learning task for which the output can be used for.                                                                                                          |
+| number_of_classes        | integer                 | Number of classes.                                                                                                                                                                 |
```

Comment:
This can be renamed to a nicer `class_count`. The old name was preserved for backward compatibility, but I never liked it.

Reply:
class_count might be interpreted as the counts of membership to a particular class (how many pixels are tree cover vs urban cover). I could see class_count being more ambiguous than number_of_classes, what do you think?

Reply:
Fair point. Fine to leave it as is.